Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView.swift

```diff
@@ -5,44 +5,51 @@
 import StreamChat
 import UIKit
 
-// TODO: Audit
-public class ChatMessageListTitleView<ExtraData: ExtraDataTypes>: UIView, UIConfigProvider {
-    public var title: String? {
-        get { titleLabel.text }
-        set { titleLabel.text = newValue }
+/// A view that is used as a wrapper for status data in navigationItem's titleView
+open class _ChatMessageListTitleView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
+    /// Content of the view that contains title (first line) and subtitle (second nil)
+    open var content: (title: String?, subtitle: String?) = (nil, nil) {
+        didSet {
+            updateContent()
+        }
     }
     
-    public var subtitle: String? {
-        get { subtitleLabel.text }
-        set { subtitleLabel.text = newValue }
-    }
+    /// Label that represents the first line of the final titleView
+    open private(set) var titleLabel: UILabel = UILabel()
+    
+    /// Label that represents the second line of the final titleView
+    open private(set) var subtitleLabel: UILabel = UILabel()
     
-    private weak var titleLabel: UILabel!
-    private weak var subtitleLabel: UILabel!
+    /// A view that joins the `titleLabel` and `subtitleLabel` in vertical stack
+    open private(set) var stackView: UIStackView = UIStackView()
     
-    public init() {
-        super.init(frame: .zero)
+    override public func defaultAppearance() {
+        super.defaultAppearance()
         
-        let titleLabel = UILabel()
         titleLabel.textAlignment = .center
         titleLabel.font = uiConfig.font.headlineBold
-        self.titleLabel = titleLabel
-
-        let subtitleLabel = UILabel()
+        
         subtitleLabel.textAlignment = .center
         subtitleLabel.font = uiConfig.font.caption1
         subtitleLabel.textColor = uiConfig.colorPalette.subtitleText
-        self.subtitleLabel = subtitleLabel
-
-        let titleView = UIStackView(arrangedSubviews: [titleLabel, subtitleLabel])
-        titleView.axis = .vertical
-        addSubview(titleView)
-        titleView.translatesAutoresizingMaskIntoConstraints = false
-        titleView.pin(to: self)
+        
+        stackView.axis = .vertical
     }
     
-    @available(*, unavailable)
-    public required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        stackView.addArrangedSubview(titleLabel)
+        stackView.addArrangedSubview(subtitleLabel)
+        addSubview(stackView)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
We have this cool `withoutAutoresizingMaskConstraints` property on `UIView` and it's subclasses, so you can use it on each views in this class, you can check other `_View`, `_Button`, `_Control` subclasses how it's done :)